WORKSPACE

```diff
@@ -46,6 +46,17 @@ kt_register_toolchains()
 load("@vaticle_dependencies//builder/python:deps.bzl", python_deps = "deps")
 python_deps()
 
+# Load //builder/rust
```

Comment:
We add the Rust toolchain and fetch crate dependencies of the gRPC + Protobuf Rust compiler.